plugins/inputs/clickhouse/README.md

```diff
@@ -0,0 +1,50 @@
+# Telegraf Input Plugin: ClickHouse
+
+This plugin gathers the statistic data from [ClickHouse](https://github.com/yandex/ClickHouse)  server.
+
+### Configuration
+```
+# Read metrics from one or many ClickHouse servers
+[[inputs.clickhouse]]
+  dsn     = "native://localhost:9000?username=user&password=qwerty"
```

Comment:
Recommend updating this to specify that a database reference is required.  I built off of this branch and ran into some issues with getting the plugin to work without specifying a database, but didn't get an obvious error (either silent failure or `driver: bad connection`). This might be obvious for people who are familiar with Clickhouse but I only maintain the TICK stack so it wasn't immediately obvious to me. I looked up the documentation [here](https://clickhouse.yandex/docs/en/interfaces/cli/) to figure out what else may have been needed, ended up just the DB name reference.